.circleci/config.yml

```diff
@@ -130,6 +130,49 @@ commands:
             - slack/status:
               channel: eng-react-integration-status
 
+  run-benchmark:
+    parameters:
+      working_directory:
+        type: string
+      NUM_PAGES:
+        type: string
+      BENCHMARK_CONTENT_SOURCE:
+        type: string
+      BENCHMARK_REPO_NAME:
+        type: string
+        default: gatsbyjs/gatsby
+      BENCHMARK_SITE_TYPE:
+        type: string
+        default: BLOG
+    steps:
+      - checkout
+      - run:
+          command: npm install
+          working_directory: << parameters.working_directory >>
+      - run:
+          command: npm run build
+          working_directory: << parameters.working_directory >>
+          environment:
+            BENCHMARK_BUILD_TYPE: COLD_START
+            NUM_PAGES: << parameters.NUM_PAGES >>
+            BENCHMARK_CONTENT_SOURCE: << parameters.BENCHMARK_CONTENT_SOURCE >>
+            BENCHMARK_REPO_NAME: << parameters.BENCHMARK_REPO_NAME >>
+            BENCHMARK_SITE_TYPE: << parameters.BENCHMARK_SITE_TYPE >>
+            CI_NAME: circleci
+      - run:
+          command: npm install
+          working_directory: << parameters.working_directory >>
```

Comment:
is deps installation needed after first run?

Reply:
Probably not technically, but I've left it in to be consistent with all of the other benchmarks we have setup. 

If it causes any issues, we can remove it everywhere.

Reply:
It's fine - just was checking of not lost copy pasta ;)